host/tests/dsClasses/dsVideoDevice.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import yaml
+import os
+import sys
+from enum import Enum, auto
+import re
+
+# Add parent outside of the class directory
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../")
+
+from raft.framework.plugins.ut_raft.configRead import ConfigRead
+from raft.framework.plugins.ut_raft.utSuiteNavigator import UTSuiteNavigatorClass
+from raft.framework.plugins.ut_raft.interactiveShell import InteractiveShell
+
+
+class dsVideoDeviceClass():
+
+    moduleName = "dsVideoDevice"
+    menuConfig =  dir_path + "/dsVideoDevice_test_suite.yml"
```

Comment:
please ensure this file is fully populated with all menu items, not just L3 ones.

Reply:
done